### PR TITLE
Added description of sorting by ASCII characters to address OKTA-595653.

### PR DESCRIPTION
**Pull request by timhowie-okta**
## Description:
- **What's changed?** Added a description stating how sorting is performed by characters (ASCII sort order)
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-595653](https://oktainc.atlassian.net/browse/OKTA-595653)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 1]
```diff
@@ -1156,7 +1156,7 @@ The first three parameters in the table below correspond to different ways to li
 | limit       | Specifies the number of results returned (maximum 200).                           | Query        | Number     | FALSE    |
 | after       | Specifies the pagination cursor for the next page of users.                                                                                     | Query        | String     | FALSE    |
 | sortBy      | Specifies field to sort by (for search queries only).                                                                                           | Search query | String     | FALSE    |
-| sortOrder   | Specifies sort order asc or desc (for search queries only).                                                                                     | Search query | String     | FALSE    |
+| sortOrder   | Specifies sort order asc or desc (for search queries only). Sorting is done in ASCII sort order (that is, by ASCII character value). Sorting is case insensitive.                                                                                                                                                    | Search query | String     | FALSE    |
```
Comment (chenningzhang-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:1159]: Maybe it's just me, the sentence seem a little repetitive - maybe "Sorting is done in ASII ..., and is case insensitive" or similar?
No more comment on the correctness though, I'll leave the detail up to you.

**Review by chenningzhang-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 2]
```diff
@@ -1156,7 +1156,7 @@ The first three parameters in the table below correspond to different ways to li
 | limit       | Specifies the number of results returned (maximum 200).                           | Query        | Number     | FALSE    |
 | after       | Specifies the pagination cursor for the next page of users.                                                                                     | Query        | String     | FALSE    |
 | sortBy      | Specifies field to sort by (for search queries only).                                                                                           | Search query | String     | FALSE    |
-| sortOrder   | Specifies sort order asc or desc (for search queries only).                                                                                     | Search query | String     | FALSE    |
+| sortOrder   | Specifies sort order asc or desc (for search queries only). Sorting is done in ASCII sort order (that is, by ASCII character value), but is case insensitive.                                                                                                                                                    | Search query | String     | FALSE    |
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:1159]: Feedback: my eye read this as "but is case sensitive". If you make this change, I think it's less likely that readers make the same mistake.
```suggestion
| sortOrder   | Specifies sort order asc or desc (for search queries only). Sorting is done in ASCII sort order (that is, by ASCII character value), but isn't case sensitive.                                                                                                                                                    | Search query | String     | FALSE    |
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM

Small suggested edit

**Pull request merged**